Changes to quick-start guide (#1366)
econ-ark/HARK: Documentation/guides/quick_start.md

## Installing HARK

> HARK is an open source project that is compatible with Python 3. Currently, we recommend using version 3.10 or lower.

Reviewer: ```suggestion
HARK is an open source project that is compatible with Python 3.
We recommend using Python 3.10 or earlier with HARK.
```

The simplest way to install HARK is to use [pip](https://pip.pypa.io/en/stable/installation/).

> Before installing HARK, we recommend creating a new virtual environment, which isolates the installation of `econ-ark` from the installations of any other Python tools and packages, thus avoiding conflicts.

Reviewer: ```suggestion
Before installing HARK, we recommend creating a new 'environment',
which isolates any other installed Python tools and packages, avoiding dependency conflicts.
```

> To create a new virtual environment, enter this in your command line:

Reviewer: I think we ought recommend `conda`, perhaps replace the rest of this sub-section with the following?

````markdown
To create a new environment, follow the steps below in your command line:

1. We first set up a Conda 'environment' for HARK.
   This is a way of isolating the installation of HARK so that it does not interfere
   with any other Python scripts or modules on your computer.
   To do so, at a command line type:

   ```console
   $ conda create -n econ-ark "python=3.10"
   ```

2. Activate the environment:

   ```console
   $ conda activate econ-ark
   ```

3. Install HARK into the environment:

   ```console
   (econ-ark) $ conda install econ-ark
   ```
````

I'd also consider remove the line about installing a non-default release, I'm unsure if it fits in the quick-start guide.

```
cd [directory where you want to store the econ-ark virtual environment]
python -m venv econ-ark
```

On Windows, type:

```
econ-ark\Scripts\activate
```

On Mac or Linux:

```
source econ-ark/bin/activate
```

Then use `pip install econ-ark` to install HARK in this environment.

Alternatively, if you are using either the [Anaconda](https://anaconda.com/) distribution or [Miniconda](https://docs.conda.io/projects/miniconda/en/latest/), you can create a virtual environment using `conda`, which eliminates the need to manage the environment's location:

```
conda create -n econ-ark python=3.10
conda activate econ-ark
pip install econ-ark
```

(If you want to install a release that is not the default stable release, for instance if you want to install a development release, you'll need to use a "pinned" release number: `pip install econ-ark==0.10.1.dev1`, substituting "0.10.1.dev1" for your desired release number.)

> ### Demonstrations on using HARK

Reviewer: This sub-section should go within "Learning HARK" below

> ## Making changes to HARK

Reviewer: Arguably this doesn't belong in the quick-start guide -- perhaps just redirect to the contributing guide?
